target/linux/generic/files/drivers/net/phy/rtl8367b.c

```diff
@@ -1540,14 +1571,19 @@ static int rtl8367b_detect(struct rtl8366_smi *smi)
 			"chip mode");
 		return ret;
 	}
-
-	switch (chip_ver) {
+	if (chip_num == 0x6367 || chip_num == 0x0597 || chip_num == 0x0276) {
```

Comment:
This line better fits here:
`dev_info(smi->parent, "checking chip_num=0x%x ver=0x%x...\n", chip_num, chip_ver);`